Root/Backend/src/controller/stock-price.js

```diff
@@ -0,0 +1,27 @@
+import dotenv from "dotenv";
+import "isomorphic-fetch"
+dotenv.config({ silent: process.env.NODE_ENV === 'production' });
+
+
+const fetchCaller = (url, res, stock) => {
```

Comment:
I don't think it really matters. But why use const for function instead of func?

Reply:
lambda functions are cool.